src/test/java/com/conveyal/gtfs/GTFSTest.java

```diff
@@ -390,18 +436,30 @@ private File exportGtfs(String namespace, DataSource dataSource, boolean fromEdi
         return tempFile;
     }
 
+    private class ValuePair {
+        private final Object expected;
+        private final Object found;
+        private ValuePair (Object expected, Object found) {
+            this.expected = expected;
+            this.found = found;
+        }
+    }
+
     /**
      * Run through the list of persistence expectations to make sure that the feed was imported properly into the
      * database.
      */
     private void assertThatImportedGtfsMeetsExpectations(
-        String dbConnectionUrl,
+        Connection connection,
         String namespace,
         PersistenceExpectation[] persistenceExpectations
     ) throws SQLException {
+        // Store field mismatches here (to provide assertion statements with more details).
+        Multimap<String, ValuePair> fieldsWithMismatches = ArrayListMultimap.create();
```

Comment:
I don't think it is necessary to include this `fieldsWithMismatches` variable.  Each PersistenceExpectation iterates through all found results and if the result is not found, that causes an error.  Then for each record this code asserts the expected vs found for each field.  I don't think this provides much valuable information for telling how every record is not like the one record the test is hoping to find.  The current tests do have a small amount of records, but if a test with a much larger dataset were added, I'm not sure how it'd be useful to describe how each record is different from the expected result.  I think it's best to just say, the record in this exact form was not found.

Reply:
Why would we be looking to run this integration test with a very large dataset? I think I have to disagree on this one. Why wouldn't we want to improve the log output for a given test? If the integration test fails the PersistenceExpectation, we're leaving the maintainer with the task of combing through the expectations and the contents of the database to figure out what went wrong. It's been a couple of weeks, but I think this is exactly what happened to me while trying to add a new test.

Reply:
Ok, well yeah hopefully we can try to keep the test datasets as small as possible to make it easier to isolate issues.